receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -4,129 +4,121 @@ type: gitprovider
 sem_conv_version: 1.9.0
 
 status:
-    class: receiver
-    stability:
-        development: [metrics]
-    distributions: [liatrio]
+  class: receiver
+  stability:
+    development: [metrics]
+  distributions: [liatrio]
```

Comment:
When we're packaged into the contrib binary we need to add contrib as a distribution.